src/main/flight/pid.c

```diff
@@ -399,6 +399,8 @@ static FAST_RAM_ZERO_INIT float acErrorLimit;
 
 void pidResetITerm(void)
 {
+    extern float integratedYaw;
```

Comment:
extern should be defined in header files

Reply:
Sure. And it will be. Please consider my previous comment: 
"No need to request style changes yet since this is only an experiment for discussion."

Reply:
Not necessarily, as it allows to minimize scope